### PR TITLE
Fix Broken Links

### PR DESCRIPTION
**Pull request by Dark-Knight11**
Fixed all 404 broken links which were redirecting to Rapids repo, and replaced them with NVIDIA's repo

Thank you for contributing to ___PROJECT___ :)

Here are some guidelines to help the review process go smoothly.

1. Please write a description in this text box of the changes that are being
   made. - Fixing broken links in `Contributing.md` file

2. Please ensure that you have written units tests for the changes made/features
   added.

3. If you are closing an issue please use one of the automatic closing words as
   noted here: https://help.github.com/articles/closing-issues-using-keywords/

4. If your pull request is not ready for review but you want to make use of the
   continuous integration testing facilities please label it with `[WIP]`.

5. If your pull request is ready to be reviewed without requiring additional
   work on top of it, then remove the `[WIP]` label (if present) and replace
   it with `[REVIEW]`. If assistance is required to complete the functionality,
   for example when the C/C++ code of a feature is complete but Python bindings
   are still required, then add the label `[HELP-REQ]` so that others can triage
   and assist. The additional changes then can be implemented on top of the
   same PR. If the assistance is done by members of the rapidsAI team, then no
   additional actions are required by the creator of the original PR for this,
   otherwise the original author of the PR needs to give permission to the
   person(s) assisting to commit to their personal fork of the project. If that
   doesn't happen then a new PR based on the code of the original PR can be
   opened by the person assisting, which then will be the PR that will be
   merged.

6. Once all work has been done and review has taken place please do not add
   features or make changes out of the scope of those requested by the reviewer
   (doing this just add delays as already reviewed code ends up having to be
   re-reviewed/it is hard to tell what is new etc!). Further, please do not
   rebase your branch on master/force push/rewrite history, doing any of these
   causes the context of any comments made by reviewers to be lost. If
   conflicts occur against master they should be resolved by merging master
   into the branch used for making the pull request.

Many thanks in advance for your cooperation!

### CODE REVIEW

**Comment by GPUtester** [comment]
Can one of the admins verify this patch?

**Comment by bdice** [comment]
@Dark-Knight11 Thanks for this contribution! Can you also find-and-replace the other instances of `___PROJECT___` with `cuCollections` in the [files shown here](https://github.com/NVIDIA/cuCollections/search?q=___PROJECT___)?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
-# Contributing to ___PROJECT___
+# Contributing to _cuCollections_
```
Comment (bdice) [inline CONTRIBUTING.md]:
```suggestion
# Contributing to cuCollections
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
 
-If you are interested in contributing to ___PROJECT___, your contributions will fall
+If you are interested in contributing to ___cuCollections___, your contributions will fall
```
Comment (bdice) [inline CONTRIBUTING.md]:
```suggestion
If you are interested in contributing to cuCollections, your contributions will fall
```

**Review by bdice — COMMENTED** [review]
A couple small formatting suggestions.

**Comment by PointKernel** [comment]
@Dark-Knight11 Looks great to me. Thanks!

@bdice Any final comments?

**Review by bdice — APPROVED** [review]
@PointKernel No other comments from me — thanks @Dark-Knight11!

**Review by PointKernel — APPROVED** [review]

**Pull request merged**